src/registrar/templates/domain_base.html

```diff
@@ -5,17 +5,17 @@
 
 {% block content %}
 <div class="grid-container">
-  <div class="grid-row">
-    {% if not is_analyst_or_superuser or not analyst_action or analyst_action_location != domain.pk %}
-    <p class="font-body-md margin-top-0 margin-bottom-2
-              text-primary-darker text-semibold"
-    >
-      <span class="usa-sr-only"> Domain name:</span> {{ domain.name }}
-    </p>
-    {% endif %}
-  </div>
+
   <div class="grid-row grid-gap">
     <div class="tablet:grid-col-3">
+      {% if not is_analyst_or_superuser or not analyst_action or analyst_action_location != domain.pk %}
```

Comment:
I do not believe this is still needed with the new layout.

Reply:
Great catch, ty!